README.MD

```diff
@@ -0,0 +1,134 @@
+# A patched 'CouchDB docker-compose setup for Self-hosted LiveSync with Tailscale funnel' fork
```

Comment:
Very fine document!
However, this document will be a new README. Not a patch. It is okay to be more confident! Please make a revision for the expressions!